src/nl/rubensten/texifyidea/inspections/latex/LatexLabelConventionInspection.kt

```diff
@@ -67,7 +69,9 @@ open class LatexLabelConventionInspection : TexifyInspectionBase() {
             }
 
             val prefix = Magic.Command.labeled[context.name]!!
-            if (!required[0].startsWith("$prefix:")) {
+            val position = labelAnyCommands[cmd.name]?.position ?: continue
+            val label = required[position - 1]
```

Comment:
Crashes when the label is defined with index > 0, but fewer parameters are specified.

```tex
\newcommand{\secondArgumentIsLabel}[2]{Dummy \textbf{#1}\label{#2}}
\section{Broodje}\secondArgumentIsLabel{seca:broodje}
```

Reply:
fixed in commit ccc748569e2c106beca1ef217771924584d3d3a3